gradle-baseline-java/src/main/groovy/com/palantir/baseline/tasks/CheckJUnitDependencies.java

```diff
@@ -63,6 +72,17 @@ public final void validateDependencies() {
                 });
     }
 
+    @Classpath
+    public final Provider<List<Configuration>> getConfigurations() {
+        return getProject()
+                .provider(() ->
+                        getProject().getConvention().getPlugin(JavaPluginConvention.class).getSourceSets().stream()
+                                .filter(ss -> !ss.getName().equals(IGNORE_MAIN_CONFIGURATION))
```

Comment:
Can we extract this snippet up to `getSourceSets().stream().filter(...)` out to a method and reuse it in `validateDependencies` too?

Reply:
done